guides/source/active_job_basics.md

```diff
@@ -386,7 +386,7 @@ ActiveJob supports the following types of arguments by default:
 
 ### GlobalID
 
-Active Job supports GlobalID for parameters. This makes it possible to pass live
+Active Job supports [GlobalID](https://github.com/rails/globalid#global-id---reference-models-by-uri) for parameters. This makes it possible to pass live
```

Comment:
Since that heading (and its URL) might change, what do you think about linking directly to the README?

```suggestion
Active Job supports [GlobalID](https://github.com/rails/globalid/blob/master/README.md) for parameters. This makes it possible to pass live
```